### PR TITLE
[FEAT] Validation structure close #10

### LINKED ISSUE #10: [Login] Validation structure

### PR DESCRIPTION
**Pull request by marcucci-dev**

### CODE REVIEW

File: cat_image/validation_data.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    @staticmethod
+    def is_username_valid(login_data: dict):
+        return True
+
+    @staticmethod
+    def is_password_valid(login_data: dict):
+        return True
+
+    @staticmethod
+    def is_photos_number_valid():
```
Comment (isabel-lombardi) [inline cat_image/validation_data.py]: images not photos

Comment (isabel-lombardi) [inline cat_image/validation_data.py]: images not photos

Comment (marcucci-dev) [reply cat_image/validation_data.py]: OK

Comment (marcucci-dev) [reply cat_image/validation_data.py]: Ok

**Pull request merged**